### DISCUSSION TITLE
Combine the save to pivot table and inline create

### DISCUSSION THREAD

**Discussion by Tuoww** [Laravel-Backpack/community-forum#737] [Q&A (Help)]
Hi,
I have 3 models:

```php
class Region extends Model
{
    use CrudTrait;

    protected $fillable = [
        'title',
    ];

    public function weights()
    {
        return $this->belongsToMany(Weight::class, 'region_weights')
            ->withPivot('delivery_charge');
    }
}
```
,

```php
class Weight extends Model
{
    use CrudTrait;

    protected $fillable = [
        'weight',
    ];

    public function regions()
    {
        return $this->belongsToMany(Region::class, 'region_weights')
            ->withPivot('delivery_charge');
    }
}
``` 

and

```php
class RegionWeight extends Model
{
    use CrudTrait;

    protected $fillable = [
        'region_id',
        'weight_id',
        'delivery_charge',
    ];

    public function region()
    {
        return $this->belongsTo(Region::class);
    }

    public function weight()
    {
        return $this->belongsTo(Weight::class);
    }
}
```

I have a crudControllor for every single model. But I want to focus more on the Region model.

In my RegionCrudController I now have this in my createOperation:
```php
CRUD::field('weights')->type('relationship')->subfields([
            [
                'name' => 'delivery_charge',
                'label' => 'Price',
                'type' => 'text',

                'wrapper' => [
                    'class' => 'form-group col-md-3',
                ],
            ],
        ]);
```

But I want to add a inline_create on the weight relationship as well. I tried to add 'inline_create('true')' but this doesnt do anything.
If I remove the subfields thing I get the error: 'Route [weights-inline-create-save] not defined.'.

My question is: is it possible to get both the pivot thing and a inline create working?
My own guess is that I do something wrong with the belongsTo and so on.

Thanks in advance.

**Comment by tabacitu** [comment]
Hey @Tuoww ,

Unfortunately no, `inline_create` will not work with the `relationship` field with subfields, afaik. It is only meant to be used with the `relationship` field when what you see is a select2. It will open a modal, allow you to create that entity in a modal, then add it to the select2.

We do not plan on making those two work together at the moment. We see them as alternatives to each other.

Hope it helps.
Cheers!
